script/ebooks

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+
+// Generate EPUB and MOBI ebooks
+
+const fs = require('fs');
+const path = require('path');
+const exec = require('child_process').exec;
+const fastmatter = require('fastmatter');
+const mkdirp = require('mkdirp');
+const slug = require('slug');
+const unified = require('unified');
+const parse = require('remark-parse');
+const stringify = require('remark-stringify');
+const visit = require('unist-util-visit');
+const download = require('download');
+const ebrew = require('ebrew');
+
+const processor = unified()
+  .use(parse)
+  .use(replaceAsides)
+  .use(replaceCheckboxes)
+  .use(fixImages)
+  .use(downloadImages)
+  .use(stringify);
```

Comment:
Have you experimented with ways of converting the HTML to epub? I wonder if that would give better output and be easier to maintain long term.

I tried [node-html2epub](https://github.com/fabi1cazenave/node-html2epub), which has a really [simple way of declaring the contents](https://github.com/fabi1cazenave/node-html2epub/blob/master/examples/epub3.json) and will fetch content from the web, but the ebook it generates is corrupt and the project doesn't appear to be maintained.